src/ex2/item_manager.js

```diff
@@ -0,0 +1,53 @@
+import PokemonClient from "./PokemonClient.js";
+
+class ItemManager {
+	constructor() {
+		this.id = 0;
+		this.itemArray = [];
+		this.pokemonClient = new PokemonClient();
+	}
+	addListItem(item) {
+		const { id, value } = this.createId(item);
```

Comment:
deconstructing the item is useless here because you're only reconstructing it again in line 11 and not using its deconstructed variables.